integration-tests/pipelines/konflux-e2e-tests.yaml

```diff
@@ -174,7 +173,7 @@ spec:
         - name: test-name
           value: e2e-tests
         - name: oci-container
-          value: "$(params.oci-container-repo):$(context.pipelineRun.name)"
+          value: "$(params.oci-container-repo):$(context.pipelineRun.name)-coverport"
```

Comment:
Keeping the coverage data in a separate OCI artifact to avoid an issue when data is pushed to the same tag concurrently (from 2 tasks in finally block - already seen in CI)